rs/nns/governance/proto/ic_nns_governance/pb/v1/governance.proto

```diff
@@ -195,6 +195,18 @@ message BallotInfo {
   Vote vote = 2;
 }
 
+// Controls how much information non-controller and non-hot-key principals can
+// see about this neuron. Currently, if a neuron is private, recent_ballots and
+// joined_community_fund_timestamp_seconds are redacted when being read by an
+// unprivileged principal.
```

Comment:
This doesn't seem like it's true yet, should this comment mention that?

Reply:
Done.

> yet

This seems like a minor detail, because it will not last long.